wrappers/python/tests/nntile_core/test_tensor_strassen.py

```diff
@@ -0,0 +1,113 @@
+# @copyright (c) 2022-2023 Skolkovo Institute of Science and Technology
+#                           (Skoltech). All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/tests/nntile_core/test_tensor_strassen.py
+# Test for tensor::strassen<T> Python wrapper
+#
+# @version 1.0.0
+# @author Aleksandr Mikhalev
```

Comment:
Who is the author?

@Muxas May be we fill author as **NNTile Team** from now on?

Reply:
How will maintain these filed and watch for their consistency? Should one add oneself to authors during project-wide refactoring?

Reply:
Example of filling in authors and organisations was taken from reference BLAS source code. @daskol please provide examples from other well known open source project to start a discuss.